### PR TITLE
refactor: use composition api in player

### PR DESCRIPTION
**Pull request by mikelxk**
build(deps): Bump deps

### CODE REVIEW

File: src/components/DraggableBar.vue [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="playbar-clickable">
     <div :id="id" class="barbg" @mousedown="onMyMouseDown">
-      <div class="cur" :style="{ width: changingProgress ? cprogress * 100 + '%' : progress + '%' }">
+      <div
```
Comment (listen1) [inline src/components/DraggableBar.vue:4]: i'm not sure why code formatter behavior different. Some rules control whether return line in html tag?

Comment (mikelxk) [reply src/components/DraggableBar.vue:4]: I format it using prettier, it defined print width of 160 in .prettierrc which did extend the config from eslint.

File: .eslintrc.json [hunk 2]
```diff
@@ -1,18 +1,17 @@
 {
   "globals": {
-    "defineProps": "readonly",
```
Comment (listen1) [inline .eslintrc.json:3]: delete `"defineProps": "readonly",` may cause eslint error in vscode.

error info:  Environment key "vue/setup-compiler-macros" is unknown
see reference:  https://stackoverflow.com/a/69806781

**Comment by listen1** [comment]
bug: change language to english/french not working
reason: unknown. `next` branch code works fine. but pr branch will cause this problem.

File: src/i18n/index.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ const i18n = createI18n({
   // ...
 });
 export const setLocale = (language: Language): void => {
-  i18n.global.locale.value = language;
+  i18n.global.locale = language;
```
Comment (listen1) [inline src/i18n/index.ts:24]: not working when change to english/french. downgrade vue-i18n to 9.1.7 and change back to value style will solve problem.

File: .eslintrc.json (hunk 2 above)
Comment (mikelxk) [reply .eslintrc.json:3]: I test on a win and a linux environment but still can't reproduce the error. Maybe do a full clean install of eslint?

Comment (listen1) [reply .eslintrc.json:3]: reinstall and no more error notice, maybe related to cache when upgrade packages.

**Pull request merged**